### ISSUE TITLE
Popup to enter Email Address

### ISSUE THREAD

**Issue by ToasterUwU** [Fallenbagel/jellyseerr#1108]
### Description

I use exclusively the Jellyfin accounts, so when people login using that they have no Email associated to their account. 

In my experience people love getting personal direct notifications about the things they do on my Jellyseerr instance, but explaining to every single person in a 1 to 1 conversation that they can get that, they just need to go to "Settings" and enter an email address, is annoying.
This behavior is just to implicit and hidden for normal users to find and understand it from my experience.

### Desired Behavior

I would like to have an option for users who log in to get a popup for entering their Email address, maybe just the first time they do. 

Mainly I want the system to educate people about the notification via Email option and let them enter the only missing info (email address) right then and there. 
A "That sounds nice, sure here is my email" is all it should take. Or alternatively "No thanks" and then a little bit of text mentioning where the user can add the email later if they change their mind.

### Code of Conduct

- [X] I agree to follow Jellyseerr's Code of Conduct

**Comment by Fallenbagel** [comment]
We already had this. And there were multiple complaints to remove it so we removed it. And then reimplemnted it in a none invasive where you hust to go to smtp settings and enable `require email` option (no need to setup smtp) and they should have a warning to update it

**Comment by ToasterUwU** [comment]
I already tried that checkbox since I thought it was this kind of thing, but when testing it out I never saw a warning or popup anywhere..

What are the conditions for it to appear and how and where does it?

**Comment by gauthier-th** [comment]
It may be related to #900 and #1096. Let me check if I can the email popup working.

**Comment by Fallenbagel** [comment]
> I already tried that checkbox since I thought it was this kind of thing, but when testing it out I never saw a warning or popup anywhere..
> 
> What are the conditions for it to appear and how and where does it? 

The warning is non-invasive and should appear on the side panel right where the version number is. Does it not appear anymore?

**Comment by gauthier-th** [comment]
It works well for me:

![image](https://github.com/user-attachments/assets/c90e077a-45fe-4d82-a04b-97103e96494a)

**Comment by ToasterUwU** [comment]
I see, must have missed that.

That's still a little subtle for me honestly.

"Require User Email" would make me think it's required of people to add one to be able to proceed with anything, not that they get reminded in a little text box that is a easy to ignore or miss. I want to have this info be absolutely unavoidable in my instance.

So with that in mind I would like to change my feature request to having either another option for a popup or changing the behavior of the current setting to include one.

One way or another, the current setting should be renamed. It says "require", but there isn't actually a single requirement to do anything caused by this setting. It should be called "Remind users to add email" or something like that.

**Comment by ToasterUwU** [comment]
@Fallenbagel pinging because I think you might have missed my response.

**Comment by gauthier-th** [comment]
Renaming the setting would make sense.
But I think the current behavior fits quite well your initial request: a little reminder that you can add an email address, without being too intrusive.

You said:
> A "That sounds nice, sure here is my email" is all it should take. Or alternatively "No thanks" and then a little bit of text mentioning where the user can add the email later if they change their mind.

And then:
> I want to have this info be absolutely unavoidable in my instance.

**Comment by ToasterUwU** [comment]
I dont think it fits it well at all.

Sure, it is a reminder and it can be ignored if the user wants to, but this is a little box in the corner of the screen, or on mobile even in the corner of a hidden by default sidebar.
Its too unintrusive to the point of every single person who uses my instance missing it completely. I understand that annoying people is a bad thing, and its also not my goal here. But useful and somewhat important info should jump into your face at least once before being almost hidden away in a corner.
So i would like to see a popup added for reminding people to add an email, that never shows up again if they press a "No thank you" button or something like that.

A reminder is only useful if its not easy to overlook

**Comment by devanteweary** [comment]
Out of the about 20 friends/family I have on this, several have asked me if they can get notifications when a request is available.  
I then tell them to look at the warning in the bottom corner and they say oh they never noticed that before.

In other words, not a single person used that warning even though they wanted the function.

Would love the _option_ to have it pop up when they first log in saying something like "please enter email for notifications".  
- Non-intrusive notice.
- Popup notice.

**Comment by Fallenbagel** [comment]
> Out of the about 20 friends/family I have on this, several have asked me if they can get notifications when a request is available.  
> I then tell them to look at the warning in the bottom corner and they say oh they never noticed that before.
> 
> In other words, not a single person used that warning even though they wanted the function.
> 
> Would love the _option_ to have it pop up when they first log in saying something like "please enter email for notifications".  
> - Non-intrusive notice.
> - Popup notice.

Yeah that's how we had jellyseerr login flow previously until 1.4 or 1.5 Due to a majority backlash on the behavior we changed it like that

**Comment by ToasterUwU** [comment]
@Fallenbagel So is there a chance to get the popup back as an option? Default being the warning in the corner, and a simple setting to change it to popup would be everything i want here.

**Comment by Fallenbagel** [comment]
> @Fallenbagel So is there a chance to get the popup back as an option? Default being the warning in the corner, and a simple setting to change it to popup would be everything i want here.

That's the reason why this feature request is not closed?

**Comment by ToasterUwU** [comment]
Ok, just wanted to make sure since there was no explicit "Sounds good, will do" or something like that. 

Thank you

**Comment by Fallenbagel** [comment]
> Ok, just wanted to make sure since there was no explicit "Sounds good, will do" or something like that. 
> 

If the feature wasn't going to be implemented, it will probably be closed as `won't fix`.

**Comment by LeonMusCoden** [comment]
Bump! Would love if email could automatically be taken from the Jellyfin account or a more direct push to add the email to the profile
